publish-mqtt/src/main.rs

```diff
@@ -204,44 +211,61 @@ async fn requests(requests_tx: Sender<Request>, device_base: &str) -> Result<(),
     .await?;
     publish_retained(&requests_tx, format!("{}/$state", device_base), "ready").await?;
 
+    start_notify_sensors(bt_session, &connected_sensors);
+
     loop {
-        println!();
-        time::delay_for(update_period).await;
-        for device in &connected_sensors {
-            let temp_humidity = BluetoothGATTCharacteristic::new(
-                bt_session,
-                device.get_id() + "/service0021/char0035",
-            );
-            match temp_humidity.get_value() {
-                Err(e) => println!("Failed to get value from {}: {:?}", device.get_id(), e),
-                Ok(value) => {
-                    if let Some((temperature, humidity)) = decode_value(&value) {
-                        println!(
-                            "{} Temperature: {:.2}ºC Humidity: {:?}%",
-                            device.get_id(),
-                            temperature,
-                            humidity
-                        );
-
-                        let mac_address = device.get_address()?;
-                        let node_id = mac_address.replace(":", "");
-                        let node_base = format!("{}/{}", device_base, node_id);
-                        publish_retained(
-                            &requests_tx,
-                            format!("{}/temperature", node_base),
-                            &temperature.to_string(),
-                        )
-                        .await?;
-                        publish_retained(
-                            &requests_tx,
-                            format!("{}/humidity", node_base),
-                            &humidity.to_string(),
-                        )
-                        .await?;
-                    } else {
-                        println!("Invalid value from {}", device.get_id());
-                    }
-                }
+        for event in bt_session
+            .incoming(INCOMING_TIMEOUT_MS)
```

Comment:
this appears to be the timeout for blocking waiting for events over dbus. Given that we've put this in a loop{}, the value doesn't matter that much, but I feel like 10 seconds might be more appropriate, given that we are expecting events every 6 seconds.